config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: [:show]
+  post "/dishes/:id/ingredients", to: "ingredient_dishes#create", as: :ingredient_dish
+  delete "/dishes/:id/ingredients", to: "ingredient_dishes#destroy", as: :destroy_ingredient_dish
+
+  resources :chefs, only: [:show] do
+    resources :ingredients, only: [:index]
```

Comment:
Nice RESTful routes! For this one, if you wanted to direct this route to a nested controller, you could add `controller: "chefs/ingredients"` to the end of line 12